### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by xd009642**
I realised when about to try and release 0.4.0 that there was a changelog in the readme so I've just updated it :+1:

### CODE REVIEW

**Comment by xd009642** [comment]
@bluss @LukeMathWalker if either of you don't mind this should be a quick +1, only a readme change for the changelog :smile:

**Comment by bluss** [comment]
Lgtm! It's @LukeMathWalker that invites/hands over to you so I don't have the biggest say

File: README.md [hunk 1]
```diff
@@ -27,12 +27,24 @@ Please feel free to contribute new functionality! A roadmap can be found [here](
...
 ```
 
 ## Releases
 
+* **0.4.0**
+  * New functionality:
+    * Summary statistics:
+      * Weighted variance
+      * Weighted standard deviation
+  * Improvements / breaking changes:
```
[the remark this answers is not in the document]
Comment (xd009642) [reply README.md:41]: Sure, I was just following the format and ordering of the previous entries. Would you like me to apply that change to the previous ones as well?

**Comment by xd009642** [comment]
@LukeMathWalker I've moved breaking changes to top level for current release entry and all prior ones - after all consistency is always nice :smile:

**Review by LukeMathWalker — APPROVED** [review]

**Pull request merged**